lib/bridgetown-inline-svg/tag.rb

```diff
@@ -0,0 +1,62 @@
+module BridgetownInlineSvg
+  class Tag < Bridgetown::Builder
+    attr_reader :attributes
+    attr_reader :context
+    attr_reader :markup
+    attr_reader :svg_path
+
+    def build
+      liquid_tag "svg", :render
+    end
+
+    def render(markup, builder)
+      @context = builder.context
+      @markup = markup
+
+      interpolate_variables_in_markup!
```

Comment:
I moved the interpolation to run before we parse the markup, I think currently it runs after but this way seemed more efficient.